concepts/05 Widgets/RangeSlider/00 Overview.md

```diff
@@ -17,8 +20,114 @@ The following code adds a simple **RangeSlider** to your page. The **start** and
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-range-slider
+        [min]="0"
+        [max]="100"
+        [(start)]="startValue"
```

Comment:
1. Do we really need two-way binding here?
2. Why did not you specify the start and end values right in markup?

Reply:
1. Yes, `start` and `end` values indicate the position of the left and right sliders of the changing range
2. I can do it, but these values can be changed both from the code and by user in UI. Besides, it has already been written in Angular approach before my changes